docs/docs/contributing/scala2-vs-scala3.md

```diff
@@ -0,0 +1,38 @@
+# Divergences between Scala 2 and Dotty
+The following issues encountered when compiling Scala 2 code as-is under Dotty:
+
+## Scalafix candidates
+- If a method is defined `toSet()`, it cannot be called `toSet`.
+- “result type of implicit definition needs to be given explicitly”
+- There are no `'Symbol`s in Scala 3, you must construct symbols via `new Symbol("foo")` instead of old `'foo`
+
+## Trivial
+- Scala 2.13 libraries cannot be used from Dotty, because the type signatures have Scala version `5.3` but `5.0` is expected.
```

Comment:
The main reason Scala 2.13 libraries cannot be used is that dotty-library is compiled against the 2.12 standard library which is not binary-compatible with the 2.13 one, we can't be compatible with both at the same time.